docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -95,6 +106,87 @@ If you manually start or stop a sensor in Dagit, that will override any default
 
 Once your sensor is started, if you're running the [dagster-daemon](/deployment/dagster-daemon) process as part of your deployment, the sensor will begin executing immediately, without needing to restart the dagster-daemon process.
 
+## Asset sensors
+
+A useful pattern is to create a sensor that checks for new <PyObject object="AssetMaterialization" /> events for a particular asset key. This can be used to kick off a job that computes downstream assets or notifies appropriate stakeholders.
+
+One benefit of this pattern is that it enables cross-job and even cross-repository dependencies. Each job run instigated by an asset sensor is agnostic to the job that caused it.
+
+Dagster provides a special asset sensor definition format for sensors that fire a single RunRequest based on a single asset materialization. Here is an example of a sensor that generates a RunRequest for every materialization for the asset key `my_table`:
+
+```python file=/concepts/partitions_schedules_sensors/sensors/sensors.py startafter=start_asset_sensor_marker endbefore=end_asset_sensor_marker
+from dagster import AssetKey, asset_sensor
+
+
+@asset_sensor(asset_key=AssetKey("my_table"), job=my_job)
+def my_asset_sensor(context, asset_event):
+    yield RunRequest(
+        run_key=context.cursor,
+        run_config={
+            "ops": {
+                "read_materialization": {
+                    "config": {
+                        "asset_key": asset_event.dagster_event.asset_key.path,
+                    }
+                }
+            }
+        },
+    )
+```
+
+### Multi-asset sensors
+
+Multi-asset sensors, which can trigger job executions based on some combination of states from multiple asset materialization event streams, can be handled using the base sensor definition and manual cursor management based on the asset event streams. These asset event streams can be queried using the `instance` attribute off of the <PyObject object="SensorEvaluationContext" /> object.
```

Comment:
This is a pretty big, informationally-dense block of text to parse through - mind if I take a shot at breaking it up a bit? Maybe something like this:

> Multi-asset sensors can trigger job executions based on states from multiple asset materialization event streams. This sensor type can be handled using the base sensor definition and manual cursor management on the asset event streams.

> To query the asset event streams, use the `instance` attribute on the `SensorEvaluationContext` object. For example:

> [[code]]

Reply:
Sounds great! ✅